packages/tasks/src/tasks/automatic-speech-recognition/inference.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Inference code generated from the JSON schema spec in ./spec
+ *
+ * Using src/scripts/inference-codegen
+ */
+/**
+ * Inputs for Automatic Speech Recognition inference
+ */
+export interface AutomaticSpeechRecognitionInput {
+	/**
+	 * The input audio data
+	 */
+	data: unknown;
+	/**
+	 * Additional inference parameters
+	 */
+	parameters?: {
```

Comment:
`return_timestamps` is supported by a bunch of models.
For Whisper, you have many of the generative params as well

Reply:
See https://huggingface.co/docs/transformers/main/en/main_classes/pipelines#transformers.AutomaticSpeechRecognitionPipeline

Reply:
good catch, thanks!